tests/unit/test_expression_tree/test_unary_operators.py

```diff
@@ -65,15 +69,17 @@ def test_negation(self, mocker):
         }
         assert pybamm.Negate._from_json(input_json) == nega
 
-    def test_absolute(self, mocker):
+    @settings(suppress_health_check=[HealthCheck.function_scoped_fixture])
+    @given(random_value=st.floats(min_value=-1e3, max_value=1e3))
```

Comment:
```suggestion
    @given(st.floats(min_value=-1e38, max_value=0))
```
Since we are only working with negative numbers.